administrator/components/com_joomlaupdate/tmpl/upload/default.php

```diff
@@ -14,9 +14,8 @@
 use Joomla\CMS\Language\Text;
 use Joomla\CMS\Updater\Update;
 use Joomla\CMS\Utility\Utility;
-use Joomla\Component\Joomlaupdate\Administrator\View\Joomlaupdate\HtmlView;
 
-/** @var HtmlView $this */
+/** @var \Joomla\Component\Joomlaupdate\Administrator\View\Joomlaupdate\HtmlView $this */
```

Comment:
```suggestion
/** @var \Joomla\Component\Joomlaupdate\Administrator\View\Upload\HtmlView $this */
```
<img width="995" height="693" alt="image" src="https://github.com/user-attachments/assets/38290909-ab1a-4414-8aa9-226b6dc7166e" />